From 0152062cf5ea66b10f49c9ef7b32c8825c4bd9b8 Mon Sep 17 00:00:00 2001
From: l10n daemon script <scripty@kde.org>
Date: Thu, 14 Jun 2018 05:55:39 +0200
Subject: SVN_SILENT made messages (.desktop file) - always resolve ours

In case of conflict in i18n, keep the version of the branch "ours"
To resolve a particular conflict, "git checkout --ours path/to/file.desktop"
---
 src/plasma/metadata.desktop.cmake | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/plasma/metadata.desktop.cmake b/src/plasma/metadata.desktop.cmake
index 2482736..5a729b6 100644
--- a/src/plasma/metadata.desktop.cmake
+++ b/src/plasma/metadata.desktop.cmake
@@ -9,6 +9,7 @@ Name[de]=Softwareaktualisierungen
 Name[en_GB]=Software Updates
 Name[es]=Actualizaciones de software
 Name[fi]=Ohjelmistopäivitykset
+Name[fr]=Mises à jour des logiciels
 Name[gl]=Actualizacións de software
 Name[it]=Aggiornamenti software
 Name[lt]=Programų atnaujinimai
@@ -35,6 +36,7 @@ Comment[de]=Softwareaktualisierungen beziehen
 Comment[en_GB]=Get software updates
 Comment[es]=Obtener actualizaciones de software
 Comment[fi]=Nouda ohjelmistopäivitykset
+Comment[fr]=Télécharger les mises à jour des logiciels
 Comment[gl]=Obteña actualizacións de software.
 Comment[it]=Scarica gli aggiornamenti del software
 Comment[lt]=Parsisiųsti programų atnaujinimus
-- 
cgit v1.1

